src/cpdb/helpers.py

```diff
@@ -102,5 +98,12 @@ def sort_base_on_option(
     return df_sort
 
 
+def construct_url(branch, table, build="latest", sub_folder=""):
```

Comment:
I think this is a good approach/template for DRYing out how the URL is called as mentioned in #237.